docs: book pass + rpc chapter (#3070)
paradigmxyz/reth: book/installation/installation.md

Reth runs on Linux and macOS (Windows tracked).

There are three core methods to obtain Reth:

* [Pre-built binaries](./binaries.md)
* [Docker images](./docker.md)
* [Building from source.](./source.md)

## Hardware Requirements

The hardware requirements for running Reth depend on the node configuration and can change over time as the network grows or new features are implemented.

The most important requirement is by far the disk, whereas CPU and RAM requirements are relatively flexible.

|           | Archive Node                       | Full Node                           |
|-----------|------------------------------------|-------------------------------------|

> | Disk      | At least 2.5TB (NVMe recommended)  | TBD                                 |

Reviewer: ```suggestion
| Disk      | At least 2 (NVMe recommended)  | TBD                                 |
```

> * Archive Node: At least 2.5TB is required to store 

Reviewer: ```suggestion
* Archive Node: At least 2TB is required to store all historical state.
```

* Full Node: TBD

> NVMe drives are recommended for the best performance, with SSDs being a cheaper alternative. HDDs are the cheapest option, but they will take the longest to sync, and are not recommended.

Reviewer: ```suggestion
NVMe drives are recommended for the best performance, with SSDs being a cheaper alternative. HDDs are the cheapest option, but they will take the longest to sync, and are not recommended. At the time of writing, syncing an Ethereum mainnet node to block 17.4M on NVMe drives takes about 50 hours, while on a GCP "Persistent SSD" it takes ~5 days.
```
